flexget/plugins/clients/qbittorrent.py

```diff
@@ -62,31 +66,32 @@ def connect(self, config):
         'localhost'.
         """
         self.session = Session()
-        self.url = 'http://{}:{}'.format(config['host'], config['port'])
+        self.url = '{}://{}:{}'.format('https' if config['use_ssl'] else 'http', config['host'], config['port'])
         if config.get('username') and config.get('password'):
             data = {'username': config['username'],
                     'password': config['password']}
-            self._request('post', self.url + '/login', data=data, msg_on_fail='Authentication failed.')
+            self._request('post', self.url + '/login', data=data, msg_on_fail='Authentication failed.',
+                          verify=config['verify_cert'])
         log.debug('Successfully connected to qBittorrent')
         self.connected = True
 
-    def add_torrent_file(self, file_path, data):
+    def add_torrent_file(self, file_path, data, verify_cert):
         if not self.connected:
             raise plugin.PluginError('Not connected.')
         multipart_data = {k: (None, v) for k, v in data.items()}
         with open(file_path, 'rb') as f:
             multipart_data['torrents'] = f
             self._request('post', self.url + '/command/upload', msg_on_fail='Failed to add file to qBittorrent',
-                          files=multipart_data)
+                          files=multipart_data, verify=verify_cert)
```

Comment:
Pep8 alignment issues here. The start of the second line inside parenthesis should line up with the parenthesis ie.
```
self._request('post', self.url + '/command/upload', msg_on_fail='Failed to add file to qBittorrent',
                      files=multipart_data, verify=verify_cert)
```
Fix that here and on line 94 and it should be good to go.

Reply:
Also fixed it in line 74